net/pfSense-pkg-frr/files/usr/local/pkg/frr/frr_bgp_neighbors.xml

```diff
@@ -375,6 +375,48 @@
 			<source_value>value</source_value>
 		</field>
 
+		<field>
+			<name>BFD</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr></fielddescr>
+			<description>
+				<![CDATA[
+				<b>BFD Enable</b> : Listen for BFD events registered on the same target as this BGP neighbor.
+				When BFD peer goes down it immediately asks BGP to shutdown the connection with its neighbor and, when it goes back up, notify BGP to try to connect to it.
+				<br></br>
+				<b>BFD Check Control Plane Failure</b>: Allow to write CBIT independence in BFD outgoing packets. Also allow to read both C-BIT value of BFD and lookup BGP peer status.
+				This command is useful when a BFD down event is caught, while the BGP peer requested that local BGP keeps the remote BGP entries as staled if such issue is detected.
+				This is the case when graceful restart is enabled, and it is wished to ignore the BD event while waiting for the remote router to restart
```

Comment:
Reword to avoid the passive voice ("it is wished to ignore").
Should "BD" be "BFD"?

Could be shortened to "This is the case when graceful restart is enabled and BFD should ignore the event while waiting for the remote to restart".